forge-gui/res/cardsfolder/upcoming/banes_contingency.txt

```diff
@@ -0,0 +1,10 @@
+Name:Bane's Contingency
+ManaCost:1 U U
+Types:Instant
+A:SP$ Counter | TargetType$ Spell | TgtPrompt$ Select target spell | ValidTgts$ Card | ConditionCheckSVar$ X | ConditionSVarCompare$ EQ0 | StackDescription$ SpellDescription | SubAbility$ DBCounter | Counter target spell. If that spell targets a commander you control, instead counter that spell, scry 2, then draw a card.
+SVar:DBCounter:DB$ Counter | Defined$ Targeted | ConditionCheckSVar$ X | ConditionSVarCompare$ GE1 | SubAbility$ DBScry | StackDescription$ None
+SVar:DBScry:DB$ Scry | ScryNum$ 2 | SubAbility$ DBDraw
```

Comment:
Doesn't Scry + Draw always happen like that?
Ideally you only need one CounterEffect and the Conditions on the other two

Reply:
This configuration does work in testing, as it was directly modeled from other "if X then also Y" counterspell scripts such as Desertion. I did change the script to match your suggestion and it does also work, which saves a line or two. Updated the script to match. Thanks!